src/checkers/inference/InferenceDevelLauncher.java

```diff
@@ -69,6 +70,12 @@ protected void initInferenceOptions(String [] args) {
         InferenceOptions.checkerJar = new File (InferenceOptions.distDir, "checker.jar");
     }
 
+    // what used as bootclass to run the compiler
```

Comment:
This confused me for a bit... why override a method with the exact same implementation.
Then I realized that `RUNTIME_BCP` is static and bound to two different constants.
How about not introducing two separate keys for this?
It's a bit ugly that `InferenceDevelLauncher` will then use two different property prefixes, but that still seems much easier to understand than this override.